docs/advanced-usage/connecting-cloud-storage/s3-compatible.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
we might need to emphasize that the bucket name needs to respect [s3 bucket naming rules](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html) due to strict minio checking